src/STEMMUS_SCOPE.m

```diff
@@ -28,9 +28,15 @@
 
 % Load in required Octave packages if STEMMUS-SCOPE is being run in Octave:
 if exist('OCTAVE_VERSION', 'builtin') ~= 0
-    pkg load statistics;
+    disp('Loading Octave packages...');
+    pkg load statistics io;
 end
 
+disp('Create dummy landcover values...');
+ebf = repmat({'Evergreen Broadleaf Forest'}, 40, 1);
+crop = repmat({'Croplands'}, 97-40, 1);
```

Comment:
@Yunfei-Wang1993 When testing, you can modify the length of the repetitions here to fit your test data.
I had 97 timesteps worth of data, you can change `97` to whatever number of timesteps you have.